packages/merkle-mountain-range/__tests__/core.test.ts

```diff
@@ -57,6 +57,30 @@ describe("core", () => {
     expect(verifications.every((verification) => verification === true)).toBe(true);
   });
 
+  const createMmrWithValues = async (values: string[], mmrId?: string) => {
+    const store = new MemoryStore();
+    const hasher = new StarkPedersenHasher();
+    const mmr = new CoreMMR(store, hasher, mmrId);
+    for (const leaf of values) {
+      await mmr.append(leaf);
+    }
+    return mmr;
+  };
+
+  it("Update successful", async () => {
```

Comment:
Rename test name to "It should successfully perform an update"

Reply:
Add a test that checks that an update properly updated also the nodes in the DB, no need to check that the values are exact just make sure they are not the same as the previous ones.

Reply:
Test multiple cases and against a bigger tree, e.g. 1024 elements